### PR TITLE
Test scannable parser class for 1.x branch

### PR DESCRIPTION
**Pull request by PaulWay**
Improving the documentation of the Scannable parser class, and testing all its functionality.

### CODE REVIEW

File: insights/tests/test_scannable.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    log = FakeAnacondaLog(ctx)
+
+    assert hasattr(log, 'has_fcoe')
+    assert log.has_fcoe is True
+    assert hasattr(log, 'panic')
+    assert log.panic is False
+    assert hasattr(log, 'warnings')
+    assert type(log.warnings) == list
+    assert len(log.warnings) == 1
+    print log.warnings
```
Comment (xiangce) [inline insights/tests/test_scannable.py]: You might forget to remove this 'print` statement.

Comment (PaulWay) [inline insights/tests/test_scannable.py]: Yep, I did forget!  I usually use a macro to detect those print statements but forgot.  Thanks @xiangce!

**Comment by PaulWay** [comment]
Odd - how did I not get around to pushing this change up?

**Comment by xiangce** [comment]
Will merge it after it's re-submitted to the master branch, see #150.

**Pull request merged**